bls/globals.go

```diff
@@ -1,11 +1,9 @@
 package bls
 
-const MODULUS_STR = "52435875175126190479447740508185965837690552500527637822603658699938581184513"
```

Comment:
can you leave this string exported so we don't have to copy it to our prysm/geth forks?  (Or is there a better way to get this value?)